pkg/providers/tencent/tencent.go

```diff
@@ -601,6 +602,12 @@ func (p *Tencent) deleteInstance(f bool) (string, error) {
 
 // CreateCheck check create command and flags.
 func (p *Tencent) CreateCheck() error {
+	if p.UserDataPath != "" {
+		_, err := os.Stat(p.UserDataPath)
+		if os.IsNotExist(err) {
+			return fmt.Errorf("[%s] calling preflight error: failed to read user data file state %s", p.GetProviderName(), p.UserDataPath)
```

Comment:
It's better to show file is not exist error. 
```
fmt.Errorf("[%s] calling preflight error: failed to read user data file %s with error %v", p.GetProviderName(), p.UserDataPath, err)
```

Reply:
How's that?

```log
FATA[0000] [tencent] calling preflight error: file test.sh is not exist, msg: stat test.sh: no such file or directory 
```